CHANGES.md

```diff
@@ -1,6 +1,10 @@
 Changelog for singletons project
 ================================
 
+2.6
+---
+* Witness adjunction between `Data.Singletons.Sigma` and `ConstSym1` with `currySigma` and `uncurrySigma`.
```

Comment:
Move this release note under `2.5`, since that's the upcoming version that will be released next.